libsodium-sys/src/crypto_secretbox_xsalsa20poly1305.rs

```diff
@@ -16,12 +16,26 @@ extern {
         mlen: c_ulonglong,
         n: *const [u8; crypto_secretbox_xsalsa20poly1305_NONCEBYTES],
         k: *const [u8; crypto_secretbox_xsalsa20poly1305_KEYBYTES]) -> c_int;
+    pub fn crypto_secretbox_detached(
```

Comment:
I don't think that's a file that exists. Should I create it?

I would've preferred to stick with the pattern in this file and import `crypto_secretbox_detached_xsalsa20poly1305`, but libsodium doesn't define that symbol. I wasn't sure whether it was better to split up the `extern` declarations or to keep them in one place and suck up with the naming inconsistency.